posh-git.psm1

```diff
@@ -85,7 +85,7 @@ if (!$currentPromptDef -or ($currentPromptDef -eq $defaultPromptDef)) {
         }
 
         $global:LASTEXITCODE = $origLastExitCode
-        $promptSuffix
+        $ExecutionContext.SessionState.InvokeCommand.ExpandString($promptSuffix)

```

Comment:
TIL

I probably would have done something way more involved like check if `$promptSuffix` was a `[ScriptBlock]` to execute every time.